### DISCUSSION TITLE
Not shutting printer down after print

### DISCUSSION THREAD

**Discussion by turbodlr** [jamesmccannon02/OctoPrint-Tplinkautoshutdown#30] [Q&A]
I have the option set to shut printer down after print.  It is not.  The power button from toolbar the plugin created works fine to turn printer off and on.  Can you advise?

**Comment by CodingTheUnknown** [comment]
@turbodlr Have you gone into settings then TpLinkHandler then set the options. If you have can you send some screen shots please. Will help a lot

**Reply by CodingTheUnknown** [reply]
Also check that if you don't use the timelapse function you have the timelapse option unchecked

**Reply by turbodlr** [reply]
hi, 
![image](https://user-images.githubusercontent.com/79548657/109541488-195fa300-7a81-11eb-967a-5b5af401114c.png)
I have not used the timelapse function but may want to in future, will it still work if this check box is unchecked?

**Reply by turbodlr** [reply]
Plug is hs105 model

**Reply by CodingTheUnknown** [reply]
Sure just uncheck it and save for now and then when you want to use it come back in, recheck the box and click save.

**Reply by turbodlr** [reply]
I unchecked it, will do a test print and see if printer shuts down after

**Reply by turbodlr** [reply]
Ok with "time lapse" unchecked it powered printer off after print finished.. any future plan to set a buffer time so time lapse can finish, then shutdown printer?

thanks for your help too!
